activity_calendar/templates/activity_calendar/activity_page_base.html

```diff
@@ -153,6 +153,14 @@ <h1>{{ activity_moment.title }}</h1>
         </div>
         <div class="col-12 col-md-5">
             {% if request.user.is_authenticated %}
+                {% if perms.activity_moment.can_edit %}
```

Comment:
This permission does not exist in the database (it's not a Django default nor is this created in the ActivityMoment's Meta class)
I also suggest we use the term 'change' instead of 'edit' (this is done for other (Django) permissions as well)